R/net.accessor.R

```diff
@@ -18,14 +17,22 @@
 #' @return a vector or a list of vector for `get_` functions. And the Master
 #'         list object for `set_` and `add_` functions
 #'
+#' @section Core Attribute:
+#' The `append_core_attr` function initialize the attributes necessary for
```

Comment:
The documentation is not built with markdown currently (I was getting an error when I tried to convert, I'll deal with a conversation later), so please change `x` to \code{x}

Reply:
Oh ok, I updated that